west.yml

```diff
@@ -63,7 +63,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 3d2f1c7f0558d76b24afdabbaff6d36ea96db768
+      revision: pull/1852/head
```

Comment:
Please update with SHA: `58284ff0e1b85f24d1be928860e9e6eb0e86b4e5`.

Reply:
Changed :) Please merge when CI is green :)